tests/backup/backup_restore_basic_test.go

```diff
@@ -3712,3 +3713,327 @@ var _ = Describe("{DeleteS3ScheduleAndCreateNfsSchedule}", func() {
+	for _, svc := range allServices.Items {
+		if svc.Name == "portworx-service" {
+			namespacesToSkip = append(namespacesToSkip, svc.Namespace)
+		}
+	}
+
+	allNamespaces, err := k8sCore.ListNamespaces(make(map[string]string))
+	for _, namespace := range allNamespaces.Items {
+		for _, skipcase := range namespacesToSkip {
+			if skipcase == namespace.GetName() {
+				dash.VerifyFatal(namespaceNamesAge[namespace.GetName()] == oldNamespaceAge[namespace.GetName()], false, fmt.Sprintf("Namespace [%s] was restored", skipcase))
+			} else {
+				dash.VerifyFatal(namespaceNamesAge[namespace.GetName()].After(oldNamespaceAge[namespace.GetName()]), true, fmt.Sprintf("Namespace[%s] not restored", namespace.GetName()))
+			}
+		}
+	}
+}
+
+// KubeSystemAndPxNamespaceSkipOnAllNSBackup check if namespaces like kube-system and px namespace
+// are backed up while taking a backup
+var _ = Describe("{KubeSystemAndPxNamespaceSkipOnAllNSBackup}", func() {
+	var (
+		scheduledAppContexts []*scheduler.Context
+		cloudCredUID         string
+		cloudCredName        string
+		backupLocationName   string
+		backupLocationUID    string
+		backupLocationMap    map[string]string
+		schedulePolicyName   string
+		schedulePolicyUid    string
+		scheduleName         string
+		appNamespaces        []string
+		backupNames          []string
+		restoreNames         []string
+		restoreName          string
+		intervalInMins       int
+		numDeployments       int
+		ctx                  context.Context
+		oldNamespaceAge      map[string]time.Time
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("KubeSystemAndPxNamespaceSkipOnAllNSBackup", "Verify if Kube-system and Px Namespace is skipped on all namespace backup", nil, 92858)
+
+		var err error
+		ctx, err = backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+
+		numDeployments = Inst().GlobalScaleFactor
+		if len(Inst().AppList) == 1 && numDeployments < 3 {
+			numDeployments = 3
+		}
+	})
+
+	It("Verify if Kube-system and Px Namespace is skipped on all namespace backup", func() {
+
+		Step("Schedule applications in destination cluster", func() {
+			log.InfoD("Scheduling applications in destination cluster")
+			err := SetDestinationKubeConfig()
+			log.FailOnError(err, "Switching context to destination cluster failed")
+			for i := 0; i < numDeployments; i++ {
+				taskName := fmt.Sprintf("dst-%s-%d", taskNamePrefix, i)
+				appContexts := ScheduleApplications(taskName)
+				for index, ctx := range appContexts {
+					appName := Inst().AppList[index]
+					ctx.ReadinessTimeout = appReadinessTimeout
+					namespace := GetAppNamespace(ctx, taskName)
+					log.InfoD("Scheduled application [%s] in destination cluster in namespace [%s]", appName, namespace)
+					appNamespaces = append(appNamespaces, namespace)
+					scheduledAppContexts = append(scheduledAppContexts, ctx)
+				}
+			}
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster failed")
```

Comment:
This might be redundant.

Reply:
I have just added it as a fail safe to avoid subsequent cases to fail due to switched kubeconfigs.

Reply:
Makes sense but AfterSuite will take care of it, right?

Reply:
Done